iOS_SDK/OneSignalSDK/build_fat_framework.sh

```diff
@@ -72,5 +65,9 @@ cd Versions
 ln -s A Current
 
 # Copy the built product to the final destination in {repo}/iOS_SDK/OneSignalSDK/Framework
-rm -rf "${ONESIGNAL_DESTINATION_PATH}/${ONESIGNAL_OUTPUT_NAME}.framework"
-cp -a "${FINAL_FRAMEWORK_LOCATION}" "${ONESIGNAL_DESTINATION_PATH}/${ONESIGNAL_OUTPUT_NAME}.framework"
\ No newline at end of file
+rm -rf "${WORKING_DIR}/Framework/OneSignal.framework"
+cp -a "${FINAL_FRAMEWORK}" "${WORKING_DIR}/Framework/OneSignal.framework"
+
+open ${WORKING_DIR}/Framework
```

Comment:
Let's print something to the console we so know what this open is showing us. Maybe something like `echo "Opening final release framework in Finder:${WORKING_DIR}/Framework/OneSignal.framework"`